packages/ui/src/views/profile-settings/settings-account-page.tsx

```diff
@@ -0,0 +1,28 @@
+import { NavLink, useLocation } from 'react-router-dom'
+
+import { Tabs, TabsList, TabsTrigger } from '@/components'
+import { SandboxLayout } from '@/views'
+
+function SettingsAccountPage() {
+  const location = useLocation()
+  const activeTab = location.pathname.split('/').pop() || 'general'
```

Comment:
why do we need this? 
if we define a path param in the routes, it should be available via `useParam()`

Reply:
oh, since this is a view, we can probably just pass the current value down as a prop